PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -93,6 +94,9 @@
         )
     )
 
+lhcInfoTable = cms.EDProducer("LHCInfoProducer",
```

Comment:
These  informations are stored event by event ?
They should be at least by lumi granularity or run.
This is of a general use.

Reply:
@mariadalfonso 
To make these nanoAOD tables, I think the information needs to be stored by event:
https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/interface/SimpleFlatTableProducer.h#L44-L45

Please correct me if I'm wrong or if you know of another piece of the nanoAOD framework that is only storing once per luminosityBlock.

Reply:
@jwill24 The  crossingAngle, betaStar and energy can do with the Run granularity. You can find an example on how to store in the `iRun` instead of in `iEvent` here 
https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/plugins/GenWeightsTableProducer.cc#L999-L1046

Reply:
Wait... The crossing angle and beta* were changed during runs, therefore I don't think we can afford per-run granularity. I think we should aim at per-ls one.

Reply:
@jwill24  let me know if you are trying to update with the edm::LuminosityBlock const& iLumi
otherwise since are 3 numbers and already well compressed currently with precision = 10 we can keep the event granularity